_episodes/09-instances.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "Singularity/Apptainer instances"
+teaching: 60
+exercises: 10
+questions:
+- "How can I keep my container running in the background?"
+- "What are the use cases for instances?"
+objectives:
+- "Run containers in a detached mode to keep services up."
+- "Deploy instances via definition files."
+keypoints:
+- Instances allow to setup services via Singularity images or definition files.
+- Code provided in Jupyter notebooks can be accompanied by a Singularity/Apptainer image with the environment needed for its execution, ensuring the reproducibility of the results.
+---
+
+As we have studied in previous chapters, commands such as `run` and `shell` allocate Singularity/Apptainer
+containers in the foreground, stopping any process running inside the container after logout. This behavior
+suits the use case of containers for executing interactive commands in a well-defined environment, but there
+are cases when running processes in the background is convenient. For example, when a web application like a
+Jupyter notebook is deployed inside a container, it is desired to keep the container up waiting for connections
```

Comment:
"up waiting for connections from the web browser" -> "up while it waits for connections..."?